src/py/aspen/aws/session.py

```diff
@@ -0,0 +1,16 @@
+from typing import Optional
+
+from boto3 import Session
+
+from .region import region
+
+_session: Optional[Session] = None
+
+
+def session() -> Session:
```

Comment:
since we have many different sessions (one for db, one for app, and now one for aws) i wonder if we should rename to something more specific? Like `aws_session`? Interested to hear what you think

Reply:
I used it as `aws.session()` in all the places to make it explicit.   Google's python style guide has some [recommendations](https://google.github.io/styleguide/pyguide.html#224-decision), but I don't know how to enforce that.  We'll just have to do it by convention...